go/test/endtoend/utils/utils.go

```diff
@@ -288,3 +287,23 @@ func convertToMap(input interface{}) map[string]interface{} {
 	output := input.(map[string]interface{})
 	return output
 }
+
+func GetInitDBSQL(initDBSQL string, updatedPasswords string, oldAlterTableMode string) (string, error) {
+	// Since password update is DML we need to insert it before we disable
+	// super_read_only therefore doing the split below.
+	splitString := strings.Split(initDBSQL, "# add custom sql here")
```

Comment:
As noted before, I think we can make this a more clear placeholder. For example:
```
# {{custom_sql}}
```

And would be nicer if the string was a const as well.